apps/include/opt.h

```diff
@@ -277,10 +277,27 @@ typedef struct options_st {
     const char *name;
     int retval;
     /*
```

Comment:
Should add a `*` or `-` here to prevent automated comment reformatting

Reply:
Good point, thanks. (Although I kinda doubt whether openssl-reformat-source will ever be reapplied to the source tree.)

Reply:
Fixed by 3a0c2bb926642e877255bdee8121f92b8e729ea2.